openmls/src/framing/plaintext.rs

```diff
@@ -715,17 +707,20 @@ impl MlsPlaintextTbs {
         authenticated_data: TlsByteVecU32,
```

Comment:
Converting this to variable length encoding should be mechanical as well. Let's do it in this PR as well.